### ISSUE TITLE
Multiple images for dark/light themes

### ISSUE THREAD

**Issue by ridder-mark** [MicrosoftDocs/AdaptiveCards#147]
To make sure the contrast on images works for accessibility we should be able to provide alternate assets for dark/light themes. Is there anything that can be done in 1.0 that can help us change the card image depending on the dark/light client ?

![image](https://user-images.githubusercontent.com/11198644/52332774-a4484d80-2a21-11e9-97e6-302bb1b39d1e.png)
![image](https://user-images.githubusercontent.com/11198644/52332794-ad391f00-2a21-11e9-8eeb-acc93260d1f5.png)
